### PR TITLE
Y24-342 - Pacbio run create store migration from axios to native fetch

### PR DESCRIPTION
**Pull request by stevieing**
Fixed inevitable test failures. Refactored PacbioRunIndex.spec.js

Closes #

#### Changes proposed in this pull request

- Add full run to PacbioRunFactory
- Refactor PacbioRunIndex.spec.js

#### Instructions for Reviewers

_[All PRs] - Confirm PR template filled_  
_[Feature Branches] - Review code_  
_[Production Merges to `main`]_  
 &nbsp; &nbsp; \- _Check story numbers included_  
 &nbsp; &nbsp; \- _Check for debug code_  
 &nbsp; &nbsp; \- _Check version_

### CODE REVIEW

File: tests/e2e/specs/pacbio/pacbio_runs_view.cy.js [hunk 1]
```diff
@@ -25,10 +25,13 @@ describe('Pacbio Runs view', () => {
     cy.get('#filterInput').should('be.visible')
     cy.get('#filterValue').should('be.visible')
     cy.get('#filterValue').children().and('contain', 'Name')
-    cy.get('#run-index').contains('tr', '5')
-    cy.get('#completeRun-2')
-    cy.get('#editRun-2')
-    cy.get('#generate-sample-sheet-2')
+    cy.get('@pacbioRunFactory').then((pacbioRunFactory) => {
```
Comment (BenTopping) [inline tests/e2e/specs/pacbio/pacbio_runs_view.cy.js:28]: Nice.

**Review by BenTopping — APPROVED** [review]
Looks good, nice to remove so much. 

Will pull down and do a bit of testing tomorrow.

**Comment by stevieing** [comment]
> Looks good, nice to remove so much.
> 
> Will pull down and do a bit of testing tomorrow.

No worries.

I have just found some places where more json data is being used so I still need to fix that but it is only tests.

File: tests/e2e/specs/pacbio/pacbio_run_create.cy.js [hunk 2]
```diff
@@ -12,20 +18,28 @@ describe('Pacbio Run Create view', () => {
     })
 
     // Find the pool being searched for by barcode
-    cy.intercept(
-      '/v1/pacbio/tubes?filter[barcode]=TRAC-2-22&include=pools.libraries.request,pools.requests,pools.used_aliquots.tag,libraries.used_aliquots.request,libraries.used_aliquots.tag&fields[requests]=sample_name&fields[tags]=group_id',
-      {
-        fixture: 'tractionPacbioTubeWithPool.json',
-      },
-    )
+    cy.wrap(PacbioTubeFactory({ findBy: 'pools' })).as('pacbioTubeFactoryWithPool')
```
Comment (seenanair) [inline tests/e2e/specs/pacbio/pacbio_run_create.cy.js:21]: cool 👏

**Review by seenanair — APPROVED** [review]
Great job, a big cleanup 👍 Just to mention, I haven't tested it locally.

**Pull request merged**